examples/Getting_Started/01_Single_Point_Calculation/httpx/run.py

```diff
@@ -18,6 +18,8 @@
 foldername = "output"
 base_url = os.getenv("PM_API_BASE_URL", "https://api.promethium.qcware.com")
 gpu_type = os.getenv("PM_GPU_TYPE", "a100")
+workflow_timeout = int(os.getenv("PM_WORKFLOW_TIMEOUT", 864000))
```

Comment:
If the environment variables aren't specified, best practice would be to avoid sending the parameter at all and just use the system defaults. No need to duplicate the defaults in usage scripts.